.vs/VSWorkspaceState.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Could you please remove the .vs directory and/or add it to .gitignore

Reply:
I did. 
I had to make more changes to get it working on my Android & IOS devices.
Please check my commits. I added comment when I though it was useful